internal/commands/request/request_internal.go

```diff
@@ -102,6 +102,8 @@ func runInternalPingOneRequest(uri string) (err error) {
 	}
 
 	if res.StatusCode < 200 || res.StatusCode >= 300 {
+		// Note we don't os.Exit(1) here because pingcli has executed
+		// without issue, despite a failed response to the custom request
```

Comment:
Is there work logged to add a parameter flag to make this a non-zero return code at the user's request? Strictly speaking this is a 0 return code, but the user's intended action has still failed and needs intercepting. @erikostien-pingidentity @samir-gandhi

Reply:
Nice, good that this is the default implementation. Since the idea is to model curl foundation we can add a `--fail` flag. From `curl` help:
```
 -f, --fail                  Fail fast with no output on HTTP errors
 ```
will open an enhancement

Reply:
enhancement will be slated for beta 2 or later unless you feel otherwise @patrickcping